src/animatediff/settings.py

```diff
@@ -1,6 +1,5 @@
 import json
 import logging
-from functools import lru_cache
```

Comment:
I'm not familiar with lru_cache, can this be removed at no risk? Does it change the behavior?